README.md

```diff
@@ -5,3 +5,158 @@
 - Web and Network  
 - Software Development
 - Web Development
+
+### BASIC COMPUTER TERMS
+
+#### Computer
+A computer is a device that executes a program or programs. 
+
+#### Hardware:
+Hardware consists of the physical components of your computer. If you can kick it, punch it, or pick it up to throw it, it’s hardware. For e.g. Mouse, Keyboard, Monitor are hardware
+
+#### Software	
+A software is a program application; which provides the computer with step-by-step instructions to perform a specific task. A computer cannot function without software.
```

Comment:
Examples can be any installable or default programs of the OS(Widows/Linux/Mac)
Installable programs applications can be games, Microsoft office and Google Chrome
Default program applications are by default installed in OS like TextEdit/Notepad, Finder/File Explorer, System Preferences/Settings. Some defaults programs are usually running in the background to support all these programs.